index.html

```diff
@@ -1606,6 +1641,282 @@ <h2>
+        HTTP Method
+      </th>
+      <th>
+        [=extension command URI Template|URI Template=]
+      </th>
+    </tr>
+    <tr>
+      <td>
+        POST
+      </td>
+      <td>
+        /session/{session id}/pressuresource/{type}
+      </td>
+    </tr>
+  </table>
+  <p>
+    This [=extension command=] allows updating the state of a [=virtual pressure source=] by pushing
+    a new [=pending sample=].
+  </p>
+  <aside class="note" data-cite="WEBDRIVER2">
+    Additionally, the [=extension command=] can be used to check whether a source type is created
+    via the [[[#create-virtual-pressure-source]]] [=extension command=] by checking the response.
+    If it has not been created the response will contain the [=error=] with
+    [=error code|WebDriver error code=]
+    <a data-cite="webdriver2/#dfn-unsupported-operation">unsupported operation</a>.
+  </aside>
+  <table class="data">
+    <caption>
+      Properties of the parameters argument used by this algorithm
+    </caption>
+    <tr>
+      <th>
+        Parameter name
+      </th>
+      <th>
+        Value type
+      </th>
+      <th>
+        Required
+      </th>
+    </tr>
+    <tr>
+      <td>
+        sample
+      </td>
+      <td>
+        {{PressureState}}
+      </td>
+      <td>
+        yes
+      </td>
+    </tr>
+  </table>
+  <p>
+    The [=remote end steps=] given |session|, |URL variables| and |parameters| are:
+  </p>
+  <ol class="algorithm" data-cite="WEBDRIVER2">
+    <li>
+      Let |virtualPressureSourceType| be the value of the |URL variables|["type"].
+    </li>
+    <li>
+      If |virtualPressureSourceType| is not a [=valid source type=], return [=error=] with [=error code|WebDriver error code=] [=invalid argument=].
+    </li>
+    <li>
+      Let |topLevelTraversable| be the <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+      context</a>'s [=browsing context/top-level traversable=].
+    </li>
+    <li>
+      Let |topLevelVirtualPressureSourceMapping| be the |topLevelTraversable|'s [=virtual pressure source mapping=].
+    </li>
+    <li>
+      If |topLevelVirtualPressureSourceMapping| does not [=map/contain=] |virtualPressureSource|,
+      return [=error=] with [=error code|WebDriver error code=]
+      <a data-cite="webdriver2/#dfn-unsupported-operation">unsupported operation</a>.
+    </li>
+    <li>
+      Let |virtualPressureSource| be |topLevelVirtualPressureSourceMapping|[|virtualPressureSourceType|].
+    </li>
+    <li>
+      Let |sample| be the result of invoking
+      <a data-cite="!WEBDRIVER2#dfn-getting-properties">get a property</a> "sample" from |parameters|.
+    </li>
+    <li>
+      If |sample| is not of type {{PressureState}}, return [=error=] with [=error code|WebDriver error code=] [=invalid argument=].
+    </li>
+    <li>
+      Set |virtualPressureSource|'s [=pending sample=] to |sample|.
+    </li>
+    <li>
+      In an [=implementation-defined=] way, run the [=data collection=] steps.
```

Comment:
The _data collection_ algorithm takes two parameters that you need to provide here.